src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -38,13 +39,16 @@ export const postSubmit = (req: Request, res: Response) => {
   return res.redirect(config.UPDATE_CHECK_YOUR_ANSWERS_URL);
 };
 
-// With validation in place we will only have 3 choices
 const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice): string => {
   if (beneficialOwnerTypeChoices === BeneficialOwnerTypeChoice.government) {
     return config.UPDATE_BENEFICIAL_OWNER_GOV_URL;
   } else if (beneficialOwnerTypeChoices === BeneficialOwnerTypeChoice.otherLegal) {
     return config.UPDATE_BENEFICIAL_OWNER_OTHER_URL;
-  } else {
+  } else if (beneficialOwnerTypeChoices === BeneficialOwnerTypeChoice.individual) {
     return config.UPDATE_BENEFICIAL_OWNER_INDIVIDUAL_URL;
+  } else if (beneficialOwnerTypeChoices === ManagingOfficerTypeChoice.individual) {
+    return config.UPDATE_MANAGING_OFFICER_INDIVIDUAL_URL;
+  } else {
+    return config.UPDATE_MANAGING_OFFICER_CORPORATE_URL;
   }
```

Comment:
would this not be better as a `switch` statement now as it is just 4 value checks on `beneficialOwnerTypeChoices`?

Reply:
changed